ts/features/mvl/index.ts

```diff
@@ -21,6 +22,14 @@ const migrations: MigrationManifest = {
     return {
       ...mvl,
       downloads: {}
+    } as PersistedMvlState;
+  },
+  // version 3
+  // as the "downloads" section has been relocated to entities, remove it
+  "3": (state: PersistedState): PersistedMvlState => {
+    const mvl = state as PersistedMvlState;
+    return {
+      ...mvl
```

Comment:
This edit outputs a
`ERROR: Unexpected key “downloads” found in previous state received by the reducer. Expected to find one of the known reducer keys instead: “byId”, “preferences”. Unexpected keys will be ignored.`

I think that a solution is either to explicitly omit the property
`return {
      ..._.omit(mvl, "downloads")
    };`    
or to just return the state as-is, relying on the whitelist below (that removes the `downloads` property from the persisted ones - which, as a one-shot side effect, will cause the store to have the downloads section in memory until the first persistence)